homeassistant/components/statistics/manifest.json

```diff
@@ -3,7 +3,9 @@
   "name": "Statistics",
   "documentation": "https://www.home-assistant.io/integrations/statistics",
   "requirements": [],
-  "dependencies": [],
+  "dependencies": [
+      "recorder"
```

Comment:
Sorry, I see now that the recorder dependency is only optional in this integration.

https://github.com/springstan/home-assistant/blob/2a594bc081a8146b892eb6a790223ac547932155/homeassistant/components/statistics/sensor.py#L123-L124

Let's remove this and leave it as it was.

Reply:
I wonder if we can move the imports in this integration if we don't depend on recorder?

Reply:
Probably not. Should I move it back and add a comment that this import cannot be moved to the top?

Reply:
Yeah, for now that is good. One solution could be that we use the `after_dependencies` key in manifest.json to specify the recorder dependency, but then the recorder integration can't have its 3rd party requirements imported at the top. I think we can wait with that.

Reply:
Sound good to me!

Reply:
Hmmm, we will most likely run into the same issues as with the MQTT.

Reply:
Yeah, so we need to wait until the recorder moves its imports back down.

Reply:
I could do that if you want. You will just have to tell me which ones I have to move back down e.g. all sqlalchemy imports, anything else?

Reply:
I think we should do that separately and first check with other core members if that's a good idea.

Reply:
Alright, anyway thanks for the insight and reviewing my PR so quickly! :)